jenkins/hru_R_Jenkinsfile

```diff
@@ -22,6 +22,7 @@ pipeline {
                           userRemoteConfigs: [[url: 'https://github.com/usgs-makerspace/wbeep-processing']]
                         ])
         sh 'aws s3 cp --recursive s3://prod-owi-resources/resources/Application/wbeep/${TIER}/hru_shape/GF_nat_reg.gdb cache/GF_nat_reg.gdb'
+		sh 'aws s3 cp --recursive s3://prod-owi-resources/resources/Application/wbeep/${TIER}/coterminous_us . --include "*"'
```

Comment:
TIL that coterminous and conterminous are both (equivalent) words 🤔

Reply:
huh! i never heard that other term before! lol

Reply:
this version is currently in higher fashion ;) 
![image](https://user-images.githubusercontent.com/6920344/69466323-5bac1780-0d49-11ea-8230-bc97df44b1ca.png)

Reply:
![image](https://user-images.githubusercontent.com/6920344/69466352-6797d980-0d49-11ea-8a41-42064d5e1b55.png)

Reply:
Haha.  Guess I'm behind the times 👴